package.nls.json

```
@@ -18,7 +18,7 @@
     "AWS.configuration.description.onDefaultRegionMissing": "Action to take when a Profile's default region is hidden in the Explorer. Possible values:\n* `add` - shows region in the explorer\n* `ignore` - does nothing with the region\n* `prompt` - (default) asks the user what they would like to do.",
     "AWS.configuration.sam.template.depth": "The maximum subfolder depth within a workspace that the Toolkit will search for SAM Template files",
     "AWS.configuration.description.samcli.debug.attach.retry.maximum": "If the Toolkit is unable to attach a debugger, this is the number of times to retry before giving up.",
-    "AWS.configuration.description.samcli.debug.attach.timeout": "Maximum amount of time to wait for debugger to connect to a SAM Local debug run (in milliseconds)",
+    "AWS.configuration.description.samcli.debug.attach.timeout": "Maximum amount of time to wait for text from SAM while doing a Local Lambda debug (in milliseconds)",
```

"output" might be more meaningful than "text". Can also make it less verbose without losing clarity:
```suggestion
    "AWS.configuration.description.samcli.debug.attach.timeout": "Maximum time (in milliseconds) to wait for SAM output while starting a Local Lambda session",
```

---

I was thinking about how to word this, that looks a lot better